.gitpod/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM gitpod/workspace-full:commit-8fc141dbdd92030a435ead06617c6d37651d8312
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG PACKER_VERSION=1.7.8
+ARG VAULT_VERSION=1.9.1
+ARG NOMAD_VERSION=1.2.2
+ARG CONSUL_VERSION=1.10.4
+ARG PULUMI_VERSION=3.18.1
+ARG CLOUDSMITH_VERSION=0.30.1
+ARG JMESPATH_VERSION=0.10.0
+ARG AWS_SSO_UTIL_VERSION=4.25.0
+ARG AWSCLI_VERSION=2.1.39
```

Comment:
It would be good to ensure these versions are up-to-date, and matching what's in our ChromeOS setup.

(at least Pulumi and Cloudsmith should be bumped to `3.25.1` and `0.32.0`, respectively.)